ntbs-service/Pages/ClinicalDetails/Edit.cshtml.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using ntbs_service.DataAccess;
+using ntbs_service.Helpers;
+using ntbs_service.Models;
+using ntbs_service.Models.Validations;
+using ntbs_service.Pages;
+using ntbs_service.Services;
+
+namespace ntbs_service.Pages_ClinicalDetails
+{
+    public class EditModel : ValidationModel
+    {
+        private readonly INotificationService service;
+        private readonly NtbsContext _context;
+
+        public EditModel(INotificationService service, NtbsContext context)
+        {
+            this.service = service;
+            _context = context;
+        }
+
+        [BindProperty]
+        public ClinicalDetails ClinicalDetails { get; set; }
+        [BindProperty]
+        public int NotificationId { get; set; }
+
+        [BindProperty]
+        public Dictionary<SiteId, bool> NotificationSiteMap { get; set; }
+        [BindProperty]
+        public List<Site> Sites { get; set; }
+        [BindProperty]
+        public NotificationSite OtherSite { get; set; }

```

Comment:
This instance variable is used for the other site description - it needs to be bound to the full NotificationSite object, rather than just a string, so we can validate the field against the data annotation on the model.